dependencies.yaml

```diff
@@ -86,6 +86,7 @@ devel_requires:
 main_requires:
   '%assetpack_requires':
   git-core:
+  openssh-common:  # for Build Service Authentication
```

Comment:
I would prefer if we can treat that as optional dependency, not as required one. How about that?

Reply:
How's that done? I couldn't find a `recommends` or `optional` around.

Reply:
that can only be done in dist/rpm/openQA.spec